cli/src/main/kotlin/commands/ScannerCommand.kt

```diff
@@ -122,17 +122,13 @@ object ScannerCommand : CommandWithHelp() {
             ScanResultsCache.configure(it)
         }
 
-        val scanner = scannerFactory?.create(config) ?: ScanCode(config)
+        val scanner = scannerFactory.create(config)
 
         println("Using scanner '$scanner'.")
 
         val ortResult = dependenciesFile?.let {
             scanner.scanDependenciesFile(it, outputDir, downloadDir, scopesToScan.toSet())
         } ?: run {
-            require(scanner is LocalScanner) {
```

Comment:
Since we already support scanner plugins, it would be better to ignore the warning than to remove this check.

Reply:
Done.